public/docs/features.md

```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
+|<span class="text-nowrap"><i class="fa fa-pencil fa-fw"></i> **Editable**</span>|✔|✔|✔|✔|✖|
+|<span class="text-nowrap"><i class="fa fa-id-card fa-fw"></i> **Limited**</span>|✔|✔|✔|✖|✖|
+|<span class="text-nowrap"><i class="fa fa-lock fa-fw"></i> **Locked**</span>|✔|✔|✖|✔|✖|
+|<span class="text-nowrap"><i class="fa fa-umbrella fa-fw"></i> **Protected**</span>|✔|✔|✖|✖|✖|
+|<span class="text-nowrap"><i class="fa fa-hand-stop-o fa-fw"></i> **Private**</span>|✔|✖|✖|✖|✖|
 
 **Only the owner of the note can change the note's permissions.**
 
-## Embed a Note:
+### Embed a Note
+
 Notes can be embedded as follows:
 
 ```xml
 <iframe width="100%" height="500" src="https://demo.codimd.io/features" frameborder="0"></iframe>
 ```
 
-## [Slide Mode](./slide-example):
+### [Slide Mode](./slide-example)
+
 You can use a special syntax to organize your note into slides.
 After that, you can use the **[Slide Mode](./slide-example)** <i class="fa fa-tv"></i> to make a presentation.
 Visit the above link for details.
 
 To switch the editor into slide mode, set the [document type](./yaml-metadata#type) to `slide`.
 
-View
-===
-## Table of Contents:
+## View
+
+### Autogenerated Table of Contents
+
 You can look at the bottom right section of the view area, there is a _ToC_ button <i class="fa fa-bars"></i>.
 Pressing that button will show you a current _Table of Contents_, and will highlight which section you're at.
 ToCs support up to **three header levels**.
 
-## Permalink
+### Permalink
+
 Every header will automatically add a permalink on the right side.
 You can hover and click <i class="fa fa-chain"></i> to anchor on it.
 
-Edit:
-===
-## Editor Modes:
-You can look in the bottom right section of the editor area, there you'll find a button with `sublime` on it.
-When you click it, you can select 3 editor modes:
+## Edit
 
-- sublime (default)
-- emacs
-- vim
+### Editor Modes
 
-## Shortcut Keys:
-The shortcut keys depend on your selected editor mode. By default they are just like Sublime text, which is pretty quick and convenient.
-> For more information, see [here](https://codemirror.net/demo/sublime.html).
+You can look in the bottom right section of the editor area, there you'll find a button with `SUBLIME` on it.
+When you click it, you can select 3 editor modes, which will also define your shortcut keys:
 
-For emacs:
-> For more information, see [here](https://codemirror.net/demo/emacs.html).
+- [Sublime](https://codemirror.net/demo/sublime.html) (default)
+- [Emacs](https://codemirror.net/demo/emacs.html)
+- [Vim](https://codemirror.net/demo/vim.html)
 
-For vim:
-> For more information, see [here](https://codemirror.net/demo/vim.html).
+### Auto-Complete
 
-## Auto-Complete:
 This editor provides full auto-complete hints in markdown.
+
 - Emojis: type `:` to show hints.
-- Code blocks: type ` ``` ` and plus a character to show hint. <i hidden>```</i>
+- Code blocks: type ` ``` ` and plus a character to show hint.
```

Comment:
```diff
-and plus a character
+and another character
```

Reply:
```suggestion
- Code blocks: type ` ``` `, followed by another character to show syntax highlighting suggestions.
```

Reply:
What do you think?

Reply:
I like it!